docs/api/protocols.rst

```diff
@@ -22,7 +22,7 @@ entity being defined.  For example:
 
```

Comment:
TODO: I don't know what this is or why it changed, but I'm reverting to master for this file 🤷‍♀